test/recover.cpp

```diff
@@ -91,52 +107,49 @@ class basic_recover_test : public beast::unit_test::suite
     }
 
     void
-    do_recover(path_type const& path,
-        fail_counter& c, error_code& ec)
+    do_recover(
+        test_store& ts, fail_counter& c, error_code& ec)
     {
-        auto const dp = path + ".dat";
-        auto const kp = path + ".key";
-        auto const lp = path + ".log";
-        recover<xxhasher, fail_file<native_file>>(dp, kp, lp, ec, c);
+        recover<xxhasher, fail_file<native_file>>(
+            ts.dp, ts.kp, ts.lp, ec, c);
         if(ec)
             return;
+        // Verify
         verify_info info;
-        verify<xxhasher>(
-            info, dp, kp, 0, no_progress{}, ec);
+        verify<xxhasher>(info, ts.dp, ts.kp,
+            0, no_progress{}, ec);
         if(ec)
             return;
-        error_code ec2;
-        native_file::erase(dp, ec2);
-        native_file::erase(kp, ec2);
-        native_file::erase(lp, ec2);
+        ts.erase();
     }
 
     void
-    test_recover(float load_factor, std::size_t count)
+    test_recover(float load_factor, std::size_t N)
     {
-        testcase(std::to_string(count) + " inserts",
+        testcase(std::to_string(N) + " inserts",
             beast::unit_test::abort_on_fail);
-        temp_dir td;
-        auto const path = td.path();
+        test_store ts{sizeof(key_type), 256, 0.95f};
```

Comment:
Should `load_factor` be used here instead of `0.95f`?
If not, `load_factor` can be removed as a `test_recover` parameter.